### ISSUE TITLE
Is Anaconda Navigator going to work on Sierra 10.12.5?

### ISSUE THREAD

**Issue by kpmaynard** [ContinuumIO/anaconda-issues#2137]
Is there a version of this program that works? It seems that every time I upgrade, it works worse and worse. I am now at the point where it does not work at all. It appears I am running Anaconda 1.6.2 my OS is macOS 10.12.5

Please let me know if I need to find an alternative.

**Comment by goanpeca** [comment]
@kpmaynard what is the problem?

**Comment by kpmaynard** [comment]
As I indicated IT DOES NOT WORK AT ALL. Before, it started, got to the splash screen indicated some status and the exited with a message saying something like "byte indices must be integer or slices, not str". I updated using a conda update --all and now the icon bounces in the dock and then stops. No splash screen or any other visible output is produced. Very frustrating. I searched around to see if I could find logs but I did not know where to look/

**Comment by kpmaynard** [comment]
As I indicated IT DOES NOT WORK AT ALL.
Before, it started, got to the splash screen indicated some status and the
exited with a message saying something like "byte indices must be integer
or slices, not str". I updated using a conda update --all and now the icon
bounces in the dock and then stops. No splash screen or any other visible
output is produced. Very frustrating.
Even when it 'worked' before, I had to re-try about 4 or 5 times on average
for the launching window to be displayed.

I searched around to see if I could find logs but I did not know where to
look.
